integration-tests/test_connection.py

```diff
@@ -96,7 +96,7 @@ def test_http_timeout(insights_client):
         3. The command fails with a return code of 1
         4. The output mentions timeout value
     """
-    insights_client.config.http_timeout = 0.1
+    insights_client.config.http_timeout = 0.001
```

Comment:
Do this change also on the [line 108](https://github.com/RedHatInsights/insights-client/blob/be23d5ea887eb6704ff2820411caf2f2a88bf42d/integration-tests/test_connection.py#L108) in this file.